.nsprc

```diff
@@ -0,0 +1,3 @@
+{
+  "exceptions": ["https://nodesecurity.io/advisories/118"]
```

Comment:
Is this fixes? Could you remove this file?

Reply:
it will make the nsp check succeed